pyemma/msm/estimators/maximum_likelihood_msm.py

```diff
@@ -207,103 +155,7 @@ def estimate(self, dtrajs, **parms):
         MSM : :class:`pyemma.msm.MaximumlikelihoodMSM`
 
         """
-        return super(MaximumLikelihoodMSM, self).estimate(dtrajs, **parms)
-
-    def _estimate(self, dtrajs):
```

Comment:
I don't know. The standard estimator and our estimator deviate from each other several times. For instance, they use a different set of discrete trajectories for the count matrix. The standard estimator needs additional checks if the stationary distribution is fixed. In the end, different model parameters are set. I don't know if the code becomes better-structured if we split this into parts.